spk/hassio/Makefile

```diff
@@ -0,0 +1,45 @@
+SPK_NAME = hassio
+SPK_VERS = $(shell date +%Y%m%d)
+SPK_REV = $(shell git rev-parse --short HEAD)
+SPK_ICON = src/$(SPK_NAME).png
+SPK_DEPENDS = "Docker>=18.09.0-0506"
+DSM_UI_DIR = app
+
+DEPENDS =
+
+MAINTAINER = fredrike+hassio@gmail.com
+DESCRIPTION = "Open source home automation that puts local control and privacy first. Powered by a worldwide community of tinkerers and DIY enthusiasts. Note: identified USB devices will be added to the file usb_devices.txt located in the hass.io folder."
+ADMIN_PORT = 8123
+RELOAD_UI = yes
+STARTABLE = yes
+DISPLAY_NAME = Hass.io
+CHANGELOG = ""
+
+UNSUPPORTED_ARCHS = $(ARM_ARCHES) $(PPC_ARCHES) $(x86_ARCHES)
+
+HOMEPAGE   = https://www.home-assistant.io/hassio/
+LICENSE    = Apache License 2.0
+HELPURL    = https://www.home-assistant.io/docs/
+
+WIZARDS_DIR = src/wizard/
+
+SERVICE_SETUP = src/service-setup.sh
+
+#DSM UI integration
+SERVICE_PORT = 8123
```

Comment:
This port is reserved for native python "Home Assistant" package: https://github.com/SynoCommunity/spksrc/wiki/SynoCommunity-Used-Ports
Please select another free port number.

Reply:
Can we configure that `homeassistant` and `hassio` (reminding that this installs a managed homeassistant) -- essentially two paths to the same result -- are incompatible, such that one cannot be installed when the other is present, and allow re-use of the `homeassistant` port for either instance of the same project?

Reply:
Agreed. It would not make sense to install `homeassitant` and `hassio` as separate packages on the same machine. `hassio` supercedes a standalone `homeassistant` install. 
By now `homeassistant` itsef is just a fraction of the ecosystem that `hassio` manages for it. I would highly recommend against messing with its default port - the entire ecosystem defaults to it.
Given that the majority of the `homeassistant` community is migrating to `hassio` where possible, I'd even say that once the `hassio` package is available for synology, it would become the logical first choice of deployment.

Reply:
I'll ignore this..

Reply:
Excuse my ignorance: When you say _markup_ - What do you mean? Put a comment in there or are you referring to extra code or settings being added somewhere? Is there a `conflicts` section that can be defined? 
Thank you!

Reply:
@chickenandpork how do we *markup* this?

Reply:
Sorry @fredrike if I used *markup* in an unusual way; I do mean flag, attribute, or configure.

@fxstein the `INFO` field would be `install_conflict_packages` based on setting a value for `SPK_CONFLICT` in the `spk/hassio/Makefile`

https://github.com/SynoCommunity/spksrc/blob/5a67e156fd8a6378d1a8e36366cf68adcaa0521f/mk/spksrc.spk.mk#L204

Seems the relevant Synology guide is over here now: https://originhelp.synology.com/developer-guide/synology_package/INFO_optional_fields.html (See 3.2.2)

Any time you feel this process is long -- and indeed, I've felt that with spksrc  -- consider that I suggested marking up the conflict a month ago, and wasn't answered.  I would blame GitHub, it's not the best conversation UI, so we all need to keep that in mind.  As well, builds are rate-limited and take a long time, so I can understand pushing back on possible reasons to rebuild by linting every PR to its limits.

Reply:
@chickenandpork Thank you very much! We are all learning how to do this right. This is very helpful and should be an easy fix.

So basically just add something like
```
SPK_CONFLICT = homeassistant
```
to the `hassio` makefile?

and vice versa  add
```
SPK_CONFLICT = hassio
```
to the `homeassistant` makefile?

Happy to submit a PR for `homeassistant` if that is required for this change.

PS: Not complaining about the length of the process, just want to help.

Reply:
I believe the change into the hassio would be sufficient, but marking in both directions seems pedantically correct ;)  This seems a more complete solution than my PR to @fredrike's branch (except the space, the one powerful extraneous space in the `SPK_VER`)

Reply:
I would recommend keeping it a separate PR but reference this PR in it. Keeps clean boundaries around the work but links the two together through GitHub.

If we agree on this, I'd be happy to make that change, test and open a separate PR for home assistant referencing this PR.

Reply:
> Any time you feel this process is long -- and indeed, I've felt that with spksrc -- consider that I suggested marking up the conflict a month ago, and wasn't answered. I would blame GitHub, it's not the best conversation UI, so we all need to keep that in mind. As well, builds are rate-limited and take a long time, so I can understand pushing back on possible reasons to rebuild by linting every PR to its limits.

I've actually missed the latest comments. It's indeed hard to keep track of comments that just comes occasionally. 

@chickenandpork what do you think think of my suggestions for uninstall? Wrote it on my phone while commuting and have limited access to a real computer in near future (only have a windows computer at work).

Reply:
```suggestion
SERVICE_PORT = 8123
SPK_CONFLICT = homeassistant
```